controllers/runner_controller.go

```diff
@@ -878,32 +878,49 @@ func newRunnerPod(template corev1.Pod, runnerSpec v1alpha1.RunnerConfig, default
 	// When you're NOT using dindWithinRunner=true,
 	// it must also be shared with the dind container as it seems like required to run docker steps.
 	//
+	// Setting VolumeSizeLimit to zero will disable /runner emptydir mount
+	//
+	// VolumeStorageMedium defines ways that storage can be allocated to a volume: "", "Memory", "HugePages", "HugePages-<size>"
```

Comment:
Probably adding this to our README would also help users discover this feature?
I guess a sub-section of https://github.com/actions-runner-controller/actions-runner-controller#additional-tweaks would be good enough.

Reply:
https://github.com/actions-runner-controller/actions-runner-controller/pull/700